fugue-aws/src/main/java/com/symphony/oss/fugue/aws/secret/AwsSecretManager.java

```diff
@@ -160,6 +160,41 @@ public IImmutableJsonDomNode getSecret(CredentialName name) throws SecretNotFoun
     }
 }
   
+  @Override
+  public String getSecretAsString(CredentialName name) throws SecretNotFoundException
+  {
+    try
+    {
+      log_.info("FETCHING SECRET "+name.toString());
```

Comment:
Move the logging into authc-service AwsApiAssembly, we will turn this off once we have decided what to do and we don't want to have to re release fugue